src/simplewallet/simplewallet.cpp

```diff
@@ -7217,7 +7246,7 @@ bool simple_wallet::get_tx_proof(const std::vector<std::string> &args)
   {
     std::string sig_str = m_wallet->get_tx_proof(txid, info.address, info.is_subaddress, args.size() == 3 ? args[2] : "");
     const std::string filename = "monero_tx_proof";
-    if (epee::file_io_utils::save_string_to_file(filename, sig_str))
+    if (m_wallet->save_to_file(filename, sig_str, true))
```

Comment:
Why does this one (and the matching load) use the new function ? This new argument is really a "please don't do your thing".

Reply:
The point is to _only_ use the new functions. 

Imagine I'm a new developer coming to this file. I want to save stuff to a file. So I have a look around how does the rest of the code do it. And I find the old version. So I do the same thing because I have no idea there's the new one, too. And I introduce a bug because the new data is not converted to hex. It's really only about consistency.

Reply:
Sounds like rationalization, but OK :)